projects/components/src/navigation/navigation.config.ts

```diff
@@ -47,3 +47,14 @@ export const enum NavItemType {
   Divider = 'divider',
   Footer = 'footer'
 }
+
+export interface NavItemGroup {
```

Comment:
We already have a header link that kind of forms a group with the following links. Let me cherry-pick this pR and play around with it

Reply:
conversation resolved in slack